mongo/client_internal_test.go

```diff
@@ -296,3 +299,40 @@ func TestClient_ListDatabaseNames_filter(t *testing.T) {
 	require.Len(t, dbs, 1)
 	require.Equal(t, dbName, dbs[0])
 }
+
+func TestClient_ReadPreference(t *testing.T) {
+	t.Parallel()
+
+	if testing.Short() {
+		t.Skip()
+	}
+	var tags = []tag.Set{
+		tag.Set{
+			tag.Tag{
+				Name:  "one",
+				Value: "1",
+			},
+		},
+		tag.Set{
+			tag.Tag{
+				Name:  "two",
+				Value: "2",
+			},
+		},
+	}
+	baseConnString := testutil.ConnString(t)
+	cs := fmt.Sprintf(
+		"%s/?readpreference=secondary&readPreferenceTags=one:1&readPreferenceTags=two:2&maxStaleness=5",
```

Comment:
This line is causing a problem when we have a topology that uses SSL. Instead of doing this, can you use the `testutil.AddOptionsToURI` function?

```
cs := testutil.AddOptionsToURI(baseConnString.String(), "readpreference=secondary&readPreferenceTags=one:1&readPreferenceTags=two:2&maxStaleness=5")
```

Reply:
`gofmt -s` doing nothing with this file. Go version: `go version go1.9.5 linux/amd64`.
Okay, I will do it with `testutil.AddOptionsToURI`